pkg/test/e2e/default_app_catalog_tests/application_test.go

```diff
@@ -0,0 +1,408 @@
+//go:build e2e
+
+/*
+Copyright 2025 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package default_app_catalog_applications_tests
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
this is in the wrong order. You need to use [gimps](https://github.com/xrstf/gimps) to format all the import statements for verify job to succeed